packages/next/server/config-shared.ts

```diff
@@ -170,9 +193,16 @@ export type NextConfig = { [key: string]: any } & {
     outputFileTracingRoot?: string
     outputStandalone?: boolean
   }
+  /** @deprecated See https://nextjs.org/blog/next-12#output-file-tracing */
+  target?: 'server' | 'serverless' | 'experimental-serverless-trace'
+  exportPathMap?: any
```

Comment:
Lets add a type here. Seems like it could be

```suggestion
  exportPathMap?: async (defaultMap?: ExportPathMap) => ExportPathMap
```

https://github.com/vercel/next.js/blob/3667eba38559e5ccb810bade4779cda3649f1413/packages/next/export/index.ts#L322